superset-frontend/package.json

```diff
@@ -282,18 +285,19 @@
     "@types/yargs": "12 - 18",
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.8.0",
     "babel-jest": "^29.7.0",
     "babel-loader": "^9.1.3",
     "babel-plugin-dynamic-import-node": "^2.3.3",
     "babel-plugin-jsx-remove-data-test-id": "^3.0.0",
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-typescript-to-proptypes": "^2.0.0",
+    "cheerio": "^1.0.0-rc.10",
```

Comment:
enzyme tests break with the released version 1.0.0

Reply:
<img src="https://media2.giphy.com/media/kVag2hXLxG4cjFmdSM/giphy.gif"/>